.github/workflows/job_keras3_backend.yml

```diff
@@ -0,0 +1,119 @@
+name: Keras 3 OpenVINO Backend
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      container:
+        description: 'JSON to be converted to the value of the "container" configuration for the job'
+        type: string
+        required: false
+        default: '{"image": null}'
+      affected-components:
+        description: 'Components that are affected by changes in the commit defined by the Smart CI Action'
+        type: string
+        required: true
```

Comment:
I do not see it being used, I suggest removing it.

Reply:
done.